### PR TITLE
Initial description of labels and their intended usage

### PR DESCRIPTION
**Pull request by DanHeidinga**
[skip ci]

Signed-off-by: Dan Heidinga <daniel_heidinga@ca.ibm.com>

### CODE REVIEW

File: doc/processes/labels.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+* `comp:jit`
+* `comp:test`
+* `comp:doc`
+* `comp:gc`
+* `comp:build`
+* `comp:port` 
+
+
+
+Dendencies
```
Comment (0xdaryl) [inline doc/processes/labels.md]: sp:  Dependencies

File: doc/processes/labels.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+---
+OpenJ9 builds with several other projects: 'Eclipse OMR' & an 'OpenJDK
+extensions for OpenJ9' repo for each JDK level.  These labels help to 
+identify dependent changes to ensure they are merged together or in the
+correct order:
+
+* `depends:omr`
+* `depends:openjdk(8, 9, 10, 11)`
+* `depends:EclipseCQ`: Indicate a change needs a Contributor Questionaire
```
Comment (0xdaryl) [inline doc/processes/labels.md]: sp:  Questionnaire

File: doc/processes/labels.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+---
+Labels for issues that affect performance or footprint.
+
+* `perf`
+* `footprint`
+
+Documentation
+---
+Labels to indicate a PR or issues should be tagged in the release notes
+or the what's new documentation.
```
Comment (0xdaryl) [inline doc/processes/labels.md]: what's new ->  What's New

File: doc/processes/labels.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+
+* `doc:releasenote`
+* `doc:whatsnew`
+
+End User issues
+---
+To indicate an issue has been raised by and end user or effects the User-exoerience rather than being a developer-centric issue.
```
Comment (0xdaryl) [inline doc/processes/labels.md]: * an end user
* effects -> affects
* sp. : user experience

**Comment by DanHeidinga** [comment]
Thanks @0xdaryl.  I've filled the spelling errors.

**Comment by 0xdaryl** [comment]
Thanks @DanHeidinga.  Looks good to me.  Please squash commits and I'll merge.

**Comment by DanHeidinga** [comment]
Squashed.

**Pull request merged**